lux-lib/src/build/utils.rs

```diff
@@ -113,56 +107,122 @@ pub(crate) fn compile_c_files(
     let build = build
         .cargo_output(false)
         .cargo_metadata(false)
-        .cargo_debug(false)
         .cargo_warnings(false)
-        .debug(false)
+        .warnings(false)
         .files(files)
         .host(std::env::consts::OS)
-        .opt_level(3)
+        .include(&lua.include_dir)
+        .opt_level(2)
```

Comment:
question: what's the rationale for lowering the opt level? Does it cause problems on windows, was this temporarily used for easier debugging of compiler output or did it cause clashes with our default cflags (which use O2)?

Reply:
That was left over from troubleshooting.